### PR TITLE
Small fixes to reduce noise

### LINKED ISSUE #616: Center rooms are not handled by `handleExternalRoom`
The logic for handling external rooms checks for highway rooms and source keeper rooms. Center rooms are not handled at all.

### PR DESCRIPTION
**Pull request by TooAngel**
- Handle center rooms, Fixes #616
- Disable watchers, don't' know what they are good for
- Reduce complexity in prototype_room_creepbuild by extracting methods
- Decreasing observer range to reduce memory (this is just a workaround)

### CODE REVIEW

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).

File: src/prototype_room_keeper.js [hunk 1]
```diff
@@ -120,7 +120,9 @@ Room.prototype.spawnKeepersEveryTicks = function(ticks) {
   let returnValue = false;
   if (Game.rooms[this.memory.base] && Game.rooms[this.memory.base].controller) {
     if (Game.rooms[this.memory.base].controller.level >= config.keepers.minControllerLevel) {
-      this.checkForWatcher();
+      // TODO I don't know what the watcher is good for, keeper rooms need to
```
Comment (mschultheiss83) [inline src/prototype_room_keeper.js:123]: i did that part... they gain you acsess for the room, to see what is happening in that room
i send them first and the "anti" keepers and sources

File: src/prototype_room_keeper.js [hunk 2]
```diff
@@ -30,11 +30,11 @@ Room.prototype.findKeepers = function(roles) {
     return _.map(positions, updateKeepersPos);
   } else {
     // todo-msc try setup?
```
Comment (mschultheiss83) [inline src/prototype_room_keeper.js:32]: why did you left this?

**Comment by mschultheiss83** [comment]
@TooAngel Sorry for my late comments

File: src/prototype_room_keeper.js (hunk 1 above)
Comment (mschultheiss83) [reply src/prototype_room_keeper.js:123]: btw the move to the center of the room and stay there...
in my version i use them also to defend Invader (InvaderCore) neibours

Comment (TooAngel) [reply src/prototype_room_keeper.js:123]: Ok, thx. I had some rooms which were constantly sending watcher creeps to one room, and keeper room harvesting is not enabled right now.

When coming to that topic I will have a second look.

File: src/prototype_room_keeper.js (hunk 2 above)
Comment (TooAngel) [reply src/prototype_room_keeper.js:32]: :-) don't know

**Pull request merged**